src/btree/bt_delete.c

```diff
@@ -169,6 +169,7 @@ __wt_delete_page(WT_SESSION_IMPL *session, WT_REF *ref, bool *skipp)
     /* History store data are always visible. No need to check visibility. */
     if (!WT_IS_HS(session->dhandle) &&
       !__wt_txn_visible(session, addr.ta.newest_txn,
+        WT_MAX(addr.ta.newest_start_durable_ts, addr.ta.newest_stop_durable_ts),
```

Comment:
Why here we all use durable ts?

Reply:
In that flow, we are going to mark the page as fast truncated only if everything is visible and stable. Due to that reason, we use the durable timestamp as the check.

Reply:
Can we add a comment for that?

Reply:
Added a comment to explain why we use the durable timestamp for the visibility check.